src/fprime_gds/common/pipeline/files.py

```diff
@@ -41,6 +42,11 @@ def setup_file_handling(
         )
         file_decoder.register(self.__downlinker)
         distributor.register("FW_PACKET_HAND", self.__uplinker)
+        if not os.access(down_store, os.W_OK):
```

Comment:
This warning will be thrown when the `down_store` already exists but is not writeable. If down_store doesn't exist **and** cannot be created, GDS fails to start up with PermissionDenied to create down_store. Should it have the same behavior in both cases?

Reply:
Fail instead of warning here.